### PR TITLE
revert reverted merge 

### PR DESCRIPTION
**Pull request by annielovesjs**
- just approve

### CODE REVIEW

**Review by SahilTara — DISMISSED** [review]

File: src/client/src/mockData/TopNavBarData.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
 const topNavBarData: string[] = [
   "Name and Output",
-  "Frameworks",
+  "Add Frameworks",
   "Pages",
```
Comment (SahilTara) [inline src/client/src/mockData/TopNavBarData.ts]: Should it be add pages here too?

**Review by streetcoder123 — DISMISSED** [review]

File: src/client/src/mockData/TopNavBarData.ts (hunk 1 above)
Comment (annielovesjs) [reply src/client/src/mockData/TopNavBarData.ts]: done

**Review by jcoc611-microsoft — APPROVED** [review]

File: src/client/src/components/TopNavBarLink/styles.module.css [hunk 2]
```diff
@@ -21,7 +21,7 @@
   display: flex;
   align-items: center;
   text-decoration: none;
-  height: 52px;
```
Comment (jasmingoh) [inline src/client/src/components/TopNavBarLink/styles.module.css:24]: I'm comparing this PR to the one that's already merged, and the height is not the same as that
https://github.com/microsoft/WebTemplateStudio/pull/860/files#diff-139cb5b75723e18fb784ac672b5273b0L24

**Review by SahilTara — APPROVED** [review]

File: src/client/src/components/TopNavBarLink/styles.module.css (hunk 2 above)
Comment (annielovesjs) [reply src/client/src/components/TopNavBarLink/styles.module.css:24]: its fine

**Review by jasmingoh — APPROVED** [review]

**Pull request merged**